api/app/Policies/PoolCandidatePolicy.php

```diff
@@ -104,6 +104,18 @@ public function archiveApplication(User $user, PoolCandidate $poolCandidate)
         return $user->isAdmin() || $user->id === $poolCandidate->user_id;
     }
 
+    /**
+     * Determine whether the user can suspend an application
+     *
+     * @param  \App\Models\User  $user
+     * @param  \App\Models\PoolCandidate  $poolCandidate
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function suspendApplication(User $user, PoolCandidate $poolCandidate)
+    {
+        return $user->id === $poolCandidate->user_id;
+    }
+
```

Comment:
should incorporate the new permissions stuff since this is new I feel

![image](https://user-images.githubusercontent.com/40485260/223568528-76db4c69-b339-4f77-9b1e-e2a06b0f59d3.png)

Reply:
Hah, there's a merge conflict now to put it in.

Reply:
Done as part of [92c94fb](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5832/commits/92c94fb845c357f12062e7874e81dbb4c40d6aa3).